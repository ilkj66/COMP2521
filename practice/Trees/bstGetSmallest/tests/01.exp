Enter the preorder traversal of the BST: 
BST:

    6
   / \
  5   8
 /     \
2       9

bstreeGetSmallest returned: 2
