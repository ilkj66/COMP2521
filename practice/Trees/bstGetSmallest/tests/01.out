Enter the preorder traversal of the BST: 
BST:

    6
   / \
  5   8
 /     \
2       9

bstGetSmallest returned: 2
